daprdocs/content/en/reference/environment/_index.md

```diff
@@ -14,7 +14,9 @@ The following table lists the environment variables used by the Dapr runtime, CL
 | APP_PORT             | Dapr sidecar | The port your application is listening on  |
 | APP_API_TOKEN        | Your application | The token used by the application to authenticate requests from Dapr API. Read [authenticate requests from Dapr using token authentication]({{< ref app-api-token >}}) for more information. |
 | DAPR_HTTP_PORT       | Your application | The HTTP port that the Dapr sidecar is listening on. Your application should use this variable to connect to Dapr sidecar instead of hardcoding the port value. Set by the Dapr CLI run command for self-hosted or injected by the `dapr-sidecar-injector` into all the containers in the pod.                                   |
+| DAPR_HTTP_ENDPOINT   | Your application | The HTTP endpoint that the Dapr sidecar is listening on. Your application should use this variable to change the endpoint used to connect to the Dapr sidecar instead of hardcoding an alternative hostname. It is expected that this value minimally include the schema and hostname.|
```

Comment:
Suggest providing an example here to make this clear "minimally include the schema and hostname"

Reply:
Added an example indicating HTTPS.